### ISSUE TITLE
import symlink component is not working (expo)

### ISSUE THREAD

**Issue by TimothyJAveni** [react/metro#573]
**Do you want to request a *feature* or report a *bug*?**
BUG

**What is the current behavior?**
i make a component symlink to my project directory. when i try to import that symlink i got error unable to resolve

**If the current behavior is a bug, please provide the steps to reproduce and a minimal repository on GitHub that we can `yarn install` and `yarn test`.**

1. make a seperate component outside the project
2. make this component symlink in project directory
3. import this symlink component
4. you will get an error

**What is the expected behavior?**
when i import symlink component it should resolve it

**Please provide your exact Metro configuration and mention your Metro, node, yarn/npm version and operating system.**
i did not make any configuration. i am using expo

```
import TestComponent from 'path of component inside project which is actual symlink'
```

**Comment by nomi9995** [comment]
+1

**Comment by eduplus** [comment]
Is this related to #1 which as you can see from the number, has been around for a long long time?

**Comment by TimothyJAveni** [comment]
@eduplus i have seen but there is not solution for import symlink as component and contributor has locked that issue as spam. so i open issue

**Comment by eduplus** [comment]
> @eduplus i have seen but there is not solution for import symlink as component and contributor has locked that issue as spam. so i open issue

I don't see the logic to open new issues to get around of issue being locked due to spam. Adding new issues is spamming even more! Also, there was not mention of #1 in the issue, which made it look like there is no previous history on this.

Don't get me wrong, I think it is really odd that this issue is still not fixed, but spamming is not the way to get it fixed.

**Comment by TimothyJAveni** [comment]
yes, you are right i opened new issue so i get solution because they locked that issue as spam without giving solution

**Comment by nomi9995** [comment]
@TimothyJAveni I have made package for tangible symlink 
https://www.npmjs.com/package/mtsl.

```
mtsl add -s ~/source-dir -d ~/User/destination-dir
mtsl start 0
```
you can use this package whenever metro maintainers do not give a good solution

**Comment by TimothyJAveni** [comment]
this is good solution but maintainer should give good solution

**Comment by TimothyJAveni** [comment]
@nomi9995 i am using `mtsl startwithoutadd` command. it is working as same `ln  -s <src> <dest>`

```
mtsl startwithoutadd -s <src> -d <dest>
```

by the way thanks

**Comment by nomi9995** [comment]
thanks. I added this option recently

**Comment by robhogan** [comment]
We're triaging older issues. This duplicates #1. Symlinks have been supported by default since Metro 0.79.0, provided the symlink target is included in `projectRoot` or [`watchFolders`](https://metrobundler.dev/docs/configuration/#watchfolders).
